airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -213,21 +212,10 @@ def __init__(
         pod_runtime_info_envs: list[k8s.V1EnvVar] | None = None,
         termination_grace_period: int | None = None,
         configmaps: list[str] | None = None,
-        resources: dict[str, Any] | None = None,
         **kwargs,
     ) -> None:
 
-        if isinstance(resources, k8s.V1ResourceRequirements):
-            warnings.warn(
-                "Specifying resources for the launched pod with 'resources' is deprecated. "
-                "Use 'container_resources' instead.",
-                category=DeprecationWarning,
-                stacklevel=2,
-            )
-            container_resources = resources
-            resources = None
-
-        super().__init__(resources=resources, **kwargs)
+        super().__init__(**kwargs)
```

Comment:
This may cause problems for users who did not follow the deprecation and did not notice change log when upgrading since resources is parameter of BaseOperator so it will not produce the usual error for using a non-existed parameter. I don't think there is much we can do about it?

Reply:
i suppose we could keep the check, but make it raise an error now?

Reply:
Sure I'll do that

Reply:
done